src/lib/devices/GenericDevice.ts

```diff
@@ -176,7 +181,9 @@ abstract class GenericDevice {
     }
 
     async init(): Promise<void> {
-        await Promise.all(this._construction);
+        for (const promise of this._construction) {
```

Comment:
promise.all is processing all states in parallel which made debugging strange because all initializations happen in parallel. Also states for controller then create states and so more happens "in parallel". so this is just streamlining this.